### DISCUSSION TITLE
Improve scheduling workflow after the core set of wg/rg sessions have been scheduled.

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#3664] [Ideas]
(Based on conversations with @flynnliz)

After the main scheduling effort, Sessions are moved from a SchedWait to Sched state by an action called "Send Notifications".

Beyond that point, scheduling new sessions (for things like AD Office Hours) becomes cumbersome. It is hard to remember to go back and rerun "Send Notifications" and there are some new ways we are using Session (with non wg/rg groups) that the workflow is likely ignoring.

What would a better workflow look like during the period from when the bulk of the wg/rg Sessions are marked Scheduled through the meeting?

**Comment by PannawatC** [comment]
Good

**Comment by PannawatC** [comment]
The

**Comment by florespinzons-ship-it** [comment]
[](url)

**Comment by SpaceWizz1** [comment]
[Aug 26, 2025 252 PM.txt](https://github.com/user-attachments/files/21995055/Aug.26.2025.252.PM.txt)

**Comment by jennaneric480-spec** [comment]
sorry my settings and computer updates so much i cant keep up
